src/main/java/com/mulesoft/connector/github/internal/Converters/ResultConverter.java

```diff
@@ -22,19 +25,16 @@ public Result<InputStream, InputStream> buildResultIssue(InputStream attributes,
         if(content == null){
             throw new RuntimeException();
         }
-//        ByteArrayOutputStream baos = new ByteArrayOutputStream();
-//        ObjectOutputStream oos = new ObjectOutputStream(baos);
-//
-//        oos.writeObject(content);
-//
-//        oos.flush();
-//        oos.close();
-//
-//        InputStream inputStreamIssue = new ByteArrayInputStream(baos.toByteArray());
-
 
         InputStream inputStreamIssue = toInputStream(new Gson().toJson(content));
 
         return Result.<InputStream, InputStream>builder().output(inputStreamIssue).attributes(attributes).build();
     }
+
+    public Result<IssueResponse, InputStream> buildResultIssueResponse (InputStream attributes, InputStream content) throws IOException {
+        if(content == null){
```

Comment:
Try to have the same logic that `IssueConverter`. Could be more comfortable always return a null so the person which get it decide if they should throw an error or return that null.